alicloud/table_alicloud_slb_load_balancer.go

```diff
@@ -0,0 +1,336 @@
+				{Name: "address_ip_version", Require: plugin.Optional},
+				{Name: "v_switch_id", Require: plugin.Optional},
+				{Name: "vpc_id", Require: plugin.Optional},
+				{Name: "load_balancer_status", Require: plugin.Optional},
+				{Name: "address_type", Require: plugin.Optional},
+				{Name: "internet_charge_type", Require: plugin.Optional},
+			},
+		},
+		GetMatrixItemFunc: BuildRegionList,
+		Columns: []*plugin.Column{
+			{
+				Name:        "load_balancer_name",
+				Type:        proto.ColumnType_STRING,
+				Description: "The name of the CLB instance.",
+			},
+			{
+				Name:        "load_balancer_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the CLB instance.",
+			},
+			{
+				Name:        "load_balancer_status",
+				Type:        proto.ColumnType_STRING,
+				Description: "The status of the CLB instance. Valid values: inactive|active|locked.",
+			},
+			{
+				Name:        "address",
+				Type:        proto.ColumnType_IPADDR,
+				Description: "The service IP address of the CLB instance.",
+			},
+			{
+				Name:        "address_type",
+				Type:        proto.ColumnType_STRING,
+				Description: "The network type of the CLB instance. Valid values: internet|intranet.",
+			},
+			{
+				Name:        "v_switch_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the vSwitch to which the CLB instance belongs.",
+				Transform:   transform.FromField("VSwitchId"),
+			},
+			{
+				Name:        "vpc_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the virtual private cloud (VPC) to which the CLB instance belongs.",
+			},
+			{
+				Name:        "network_type",
+				Type:        proto.ColumnType_STRING,
+				Description: "The network type of the internal-facing CLB instance. Valid values: vpc|classic.",
+				Transform:   transform.FromField("ReleaseTime").Transform(transform.UnixToTimestamp),
+			},
+			{
+				Name:        "master_zone_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the primary zone to which the CLB instance belongs.",
+			},
+			{
+				Name:        "slave_zone_id",
+				Type:        proto.ColumnType_STRING,
+				Description: "The ID of the secondary zone to which the CLB instance belongs.",
+			},
+			{
+				Name:        "internet_charge_type",
+				Type:        proto.ColumnType_STRING,
+				Description: "The metering method of Internet data transfer. Valid values: paybybandwidth|paybytraffic.",
+			},
+			{
+				Name:        "create_time",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Description: "The time when the CLB instance was created. ",
+			},
+			{
+				Name:        "create_time_stamp",
+				Type:        proto.ColumnType_INT,
+				Description: "The timestamp when the instance was created.",
+			},
+			{
+				Name:        "pay_type",
+				Type:        proto.ColumnType_STRING,
+				Description: "The billing method of the CLB instance. Valid values: PayOnDemand.",
```

Comment:
Does this column always return "PayOnDemand"? Does it not have any other valid value?

Reply:
Yes, in the doc it is mentioned that there is no other valid values.
https://www.alibabacloud.com/help/en/server-load-balancer/latest/describeloadbalancers